src/redux/modules/transactionsList/selectors.js

```diff
@@ -0,0 +1,23 @@
+
+export const transactionsTotalsSelector = state =>
+  state.transactionsList.transactions;
+
+
```

Comment:
why do you have a lot of redundant empty lines here?)

Reply:
Fixed